composer.json

```diff
@@ -114,7 +115,8 @@
             "Zend_Wildfire": "packages/zend-wildfire/library/",
             "Zend_Xml": "packages/zend-xml/library/",
             "Zend_XmlRpc": "packages/zend-xmlrpc/library/"
-        }
+        },
+        "classmap": ["polyfill/ReturnTypeWillChange.php"]
```

Comment:
move this before psr-0 to avoid modifying extra line jut to add a comma

Reply:
Done